src/main/java/hudson/remoting/Response.java

```diff
@@ -82,7 +83,7 @@
      */
     @Override
     void execute(Channel channel) {
-        Request req = channel.pendingCalls.get(id);
+        Request<RSP, ? extends Throwable> req = (Request<RSP, ? extends Throwable>) channel.pendingCalls.get(id);
```

Comment:
This cast is safe because the `RSP` type parameter shares the same bounds for a `Request` and its `Response`.